packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -232,7 +257,7 @@ const TradingAppCard = ({
                         new_tab={new_tab}
                         is_buttons_disabled={
                             //For MF, we enable the button even if account is not authenticated. Rest of jurisdictions, disable the button for pending, failed and needs verification
-                            is_disabled
+                            is_disabled || server_disabled
```

Comment:
just check this in the parent level
```suggestion
                            is_disabled
```